### PR TITLE
Style Login Page

### PR DESCRIPTION
**Pull request by Pav0l**
# Description

Style login page with parts created in #28 and in master without changing the code structure.
Component responsiveness will be done in #32  

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by Pav0l** [comment]
[Trello card](https://trello.com/c/neFfVXaw/69-style-the-login-page)

**Comment by Pav0l** [comment]
[Style the login page](https://trello.com/c/neFfVXaw/69-style-the-login-page)

File: src/components/pages/Login/loginValidation.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import * as yup from 'yup';
+
+
+// login schema
+const loginSchema = yup.object().shape({
+  username: yup.string().min(5).required(),
+  password: yup.string().min(5).required()
```
Comment (AnthonyJCampbell) [inline src/components/pages/Login/loginValidation.js]: For the back-end, password should have length of at least 6.

File: src/components/pages/Login/loginValidation.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    });
+
+    const isValid = false;
+
+    return { errors, isValid };
+};
+
+
+// validate against schema
+const validateLoginForm = credential => {
```
Comment (AnthonyJCampbell) [inline src/components/pages/Login/loginValidation.js]: Awesome work! 🔥

File: src/components/pages/Login/LoginForm.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+                icon="lock"
+                iconPosition="left"
+                placeholder="Password"
+                type="password"
+                name="password"
+                value={state.password}
+                onChange={handleChange}
+              />
+
+              {state.loginIn ? (
```
Comment (AnthonyJCampbell) [inline src/components/pages/Login/LoginForm.js]: Great work with the spinner - nice touches! 👍

File: src/components/pages/Login/LoginForm.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+  return (
+    <Container>
+      <Grid
+        textAlign="center"
+        style={{ height: '85vh' }}
+        verticalAlign="middle"
+      >
+        <Grid.Column style={{ maxWidth: 300 }}>
+          {state.errors.length && (
```
Comment (AnthonyJCampbell) [inline src/components/pages/Login/LoginForm.js]: Nice thinking on having the error modal be up top. Like the implementation as well!

File: src/components/pages/Login/Login.js [hunk 5]
```diff
@@ -1,77 +1,80 @@
...
-        password: credentials.password
-      });
-      changeStatus({ error: false, loading: false });
-      setUser(receivedUser);
-      return <Redirect to="/" />;
-    } catch (err) {
-      changeStatus({ error: true, loading: false });
-    }
+  const handleChange = (e, { name, value }) => {
+    setState(prevState => ({ ...prevState, [name]: value }));
```
Comment (AnthonyJCampbell) [inline src/components/pages/Login/Login.js]: For a moment there, I thought you were doing `this.setState()`. Luckily, it's all good 😂

File: src/components/pages/Login/loginValidation.js (hunk 1 above)
Comment (Pav0l) [reply src/components/pages/Login/loginValidation.js]: Fixed, good catch! 👍

**Pull request merged**